src/test/resources/features/convertImage/image.feature

```diff
@@ -0,0 +1,283 @@
+
+  @negative
+  Scenario: Verify that if one of the mandatory field is empty, it displays an error message
+    When I send a POST request to "/convertImage" with the following form data
+      | file             | @"template/image/image.jpg"      |
+      | md5              | 1589d7f466220b7604858b1eaabdd0d2 |
+      | exportFormat     |  .bmp                            |
+      | height           |                                  |
+      | width            |   10                             |
+    Then I validate the response has the "400" status code
+    And I validate that the response body should match with "common/errorResponse.json" JSON schema
+    And I validate that the response container the following values
+      | status  | 400                     |
+      | error   | "Invalid height"        |
+
+  @negative
+  Scenario: Verify that if several mandatory fields are empty, it displays corresponding messages
+    When I send a POST request to "/convertImage" with the following form data
+      | file             | @"template/image/image.jpg"      |
+      | md5              | 1589d7f466220b7604858b1eaabdd0d2 |
+      | exportFormat     |  .bmp                            |
+      | height           |                                  |
+      | width            |                                  |
+    Then I validate the response has the "400" status code
+    And I validate that the response body should match with "common/errorResponse.json" JSON schema
+    And I validate that the response container the following values
+      | status  | 400                               |
+      | error   | "Invalid height and width"        |
+
+  @negative
+  Scenario: Verify that with different md5 an error displayed????
+    When I send a POST request to "/convertImage" with the following form data
+      | file             |  @"template/image/image.jpg"      |
+      | md5              |  1589d7f466220b7604858b1eaabdd0d2 |
+      | exportFormat     |  .png                             |
+      | name             |  test                             |
+      | changeSize       |  on                               |
+      | position         |  2,5                              |
+      | height           |  10                               |
+      | width            |  10                               |
+      | gray             |  on                               |
+      | extractMetadata  |  on                               |
+      | extractThumbnail |  on                               |
+    Then I validate the response has the "400" status code
+    And I validate that the response body should match with "common/errorResponse.json" JSON schema
+    And I validate that the response container the following values
+      | status  | 400                 |
+      | error   | "Failed in the md5" |
+
+
+  @negative
+  Scenario: Verify that the position field does not allow data that is not of type (x,y)
```

Comment:
Please consider improving this scenario title to something like: `Verify that is not possible to perform image conversion with an invalid position parameter`